### PR TITLE
Adding the beginning of the ubuntu vm builder

### PR DESCRIPTION
**Pull request by hub-cap**

### CODE REVIEW

**Comment by TimSimpsonR** [comment]
+1

File: scripts/redstack [hunk 1]
```diff
@@ -241,6 +241,79 @@ cmd_build() {
...
 ###############################################################################
+# Build
+###############################################################################
+
+generate_empty_passphrase_ssh_key() {
+    expect -c "
+spawn /usr/bin/ssh-keygen -f $HOME/.ssh/id_rsa -q
+expect \"empty for no passphrase\"
+send \n
+expect assphrase
```
Comment (ed-) [inline scripts/redstack:252]: Expect what?

Comment (hub-cap) [inline scripts/redstack:252]: HELL YES

**Comment by ed-** [comment]
expect 1

**Pull request merged**